src/scripts/main.js

```diff
@@ -2,7 +2,31 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tr = document.querySelector('tr');
 
-// write your code here
+const putLibraryOnTable = () => {
+  for (let i = 0; i < people.length; i++) {
+    const row = document.createElement('tr');
+    const objectKeys = Object.keys(people[i]);
+
+    for (let j = 0; j < 6; j++) {
+      const element = document.createElement('td');
+
+      element.textContent = people[i][objectKeys[j]];
+
+      if (j === 5) {
+        element.textContent = Math.ceil(people[i].died / 100);
+      }
+
+      if (j === 4) {
+        element.textContent = people[i].died - people[i].born;
```

Comment:
Move the calculation of the age to before the loop and store it in a variable. Directly assign the value of the variable to the 'textContent' property of the 'element' when creating the corresponding table cell.